src/dashboard/server/api/validator/job.schema.json

```diff
@@ -4,5 +4,6 @@
     "team": {
       "type": "string"
     }
-  }
+  },
+  "required": ["team"]
```

Comment:
`"required"` field breaks the current post job:
![image](https://user-images.githubusercontent.com/5781796/68055608-4f82ec00-fcae-11e9-9753-1c8bbafa7a72.png)